python/res/enkf/config/custom_kw_config.py

```diff
@@ -118,3 +120,26 @@ def convertDefinition(cls, definition):
                 value_type = 0 #str
             type_hash[key] = value_type
         return type_hash
+
+    def get_abs_path(self, file):
```

Comment:
should this function be here? not used in file...

Reply:
good catch! I used it previously, not any more though